packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -34,6 +34,12 @@ contract SystemConfig is OwnableUpgradeable, Semver {
     ///         proof to fetch this value.
     bytes32 public constant UNSAFE_BLOCK_SIGNER_SLOT = keccak256("systemconfig.unsafeblocksigner");
 
+    bytes32 public constant L1_CROSS_DOMAIN_MESSENGER_SLOT = keccak256("systemconfig.l1crossdomainmessenger");
+    bytes32 public constant L1_ERC_721_BRIDGE_SLOT = keccak256("systemconfig.l1erc721bridge");
+    bytes32 public constant L1_STANDARD_BRIDGE_SLOT = keccak256("systemconfig.l1standardbridge");
+    bytes32 public constant L2_OUTPUT_ORACLE_SLOT = keccak256("systemconfig.l2outputoracle");
+    bytes32 public constant OPTIMISM_PORTAL_SLOT = keccak256("systemconfig.optimismportal");
+
```

Comment:
Should these be [offset by 1](https://github.com/ethereum-optimism/optimism/blob/a5cdb4d081d4e91f2ff7fbb1cf50d2bf00fc6a52/packages/contracts-bedrock/contracts/universal/Proxy.sol#L10) like in [erc1967](https://eips.ethereum.org/EIPS/eip-1967)?

Reply:
Should be safe to not do this given that the contract has no possibility of arbitrary storage slot writes other than `_setAddress`, which is internal and only called in `initialize`. Might be nice to add so that the preimage of the slot value is not known in the case of a second preimage attack being discovered on sha3 + future compat with new features, though.

Reply:
It may eventually be possible for users to define inputs to things that are placed in storage, would only be an issue if a second preimage attack is found